code/src/java/pcgen/gui2/csheet/CharacterSheetPanel.java

```diff
@@ -196,15 +174,13 @@ public void run()
 						}
 						catch (Throwable e)
 						{
-							final String errorMsg = "<html><body>Unable to process sheet<br>" +
-									e + "</body></html>";
-							ByteArrayInputStream instream = new ByteArrayInputStream(errorMsg.getBytes());
-							try
-							{
+							final String errorMsg = String.format("<html><body>Unable to process sheet<br>%s</body></html>", e);
```

Comment:
I typically wrap at 120 or so for Java due to the lengthier names, but I'll change the setting for pcgen.